lib/core-common/src/types.ts

```diff
@@ -173,7 +174,10 @@ export interface Builder<Config, Stats> {
     totalTime: ReturnType<typeof process.hrtime>;
     bail: (e?: Error) => Promise<void>;
   }>;
-  build: (arg: { options: Options; startTime: ReturnType<typeof process.hrtime> }) => Promise<void>;
+  build: (arg: {
+    options: Options;
+    startTime: ReturnType<typeof process.hrtime>;
+  }) => Promise<void | Stats>;
```

Comment:
Not sure if we should make this consistent with the `start` interface -- like maybe it should return `void | { stats: Stats }` for now?